tests/AheadOfTime/Trimming/SelfContained_Trimming_Test/SelfContained_Trimming_Test.fsproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net472;net7.0</TargetFrameworks>
```

Comment:
Why net7 removed here?

Reply:
I believe that is actually replaced by the specified TFM when we do a build and publish.  I expect that net472 only is there because while working on this I built from the command line a lot, and was only interested in one output.  I will however, add net7.0 for completeness.